### ISSUE TITLE
CommentsIndentation Check false-positive in empty array declarations

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#3127]
```
vlad@vlad-N61PC-M2S ~/cli $ java -jar checkstyle-6.17-all.jar com.puppycrawl.tools.checkstyle.Main -c my_check.xml Test.java
Starting audit...
[ERROR] /home/vlad/cli/Test.java:4: Comment has incorrect indentation level 12, expected is 25, indentation should be the same level as line 3. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
vlad@vlad-N61PC-M2S ~/cli $ cat Test.java
public class Test {
    void foo3() {
        String[] array = { // line 3
            // violation - line 4
        };
    }
}
vlad@vlad-N61PC-M2S ~/cli $ cat my_check.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">


<module name = "Checker">
    <module name="TreeWalker">
         <module name="CommentsIndentation">
         </module>
    </module>
</module>
```
Expected - no violations.

**Comment by romani** [comment]
Fix is merged
